test/es-module/test-esm-loader-mock.mjs

```diff
@@ -43,3 +107,5 @@ assert.deepStrictEqual(mockedV2, Object.defineProperty({
   enumerable: false,
   value: 'Module'
 }));
+
+delete globalThis.mockedModules;
```

Comment:
Can you explain why are we deleting that? Is it simply to not trigger the check of Node.js test suite or is it related to the test? In any case, adding an explanatory comment would be nice.

Reply:
Just to get the test to pass, yes. After I did this I discovered `allowGlobals` but this already has passing CI 😀

Reply:
Using `allowGlobals` or adding a comment SGTM.

Reply:
I refactored to just export the variable and have the mocked modules import it. That avoids needing to use `globalThis`.